telemetry/definitions/commonDefinitions.json

```diff
@@ -5971,6 +5988,24 @@
                 }
             ]
         },
+        {
+            "name": "languageServer_setup",
+            "description": "LSP setup event",
+            "metadata": [
+                {
+                    "type": "languageServerResourceLocation",
+                    "required": false
+                },
+                {
+                    "type": "languageServerSetupStage",
+                    "required": true
+                },
+                {
+                    "type": "manifestVersion",
```

Comment:
`manifestVersion` seems like a necessary "specific" metric. Do we also need a `version` field which here can be used to indicate the lsp server version? Or could we just name this field `version`.

Reply:
I think VS has implemented it as "manifestSchemaVersion" because there was need of multiple version fields. such as languageServerVersion & Manifest's version.